project.clj

```diff
@@ -26,4 +26,5 @@
                                    :target :nodejs
                                    :optimizations :simple}}]}
 
-  :aliases {"test-all" ["do" "test," "doo" "node" "test" "once"]})
+  :aliases {"test-all" ["do" "test," "doo" "node" "test" "once"]
+            "test-planck" ["doo" "planck" "test" "once"]})
```

Comment:
Merge `test-planck` into `test-all` so that "all" really means all. `test-all` is run for all changes and is critical to finding regressions.

Reply:
Currently, it is a bit complicated to run test-planck on a CI environment. I have asked Mike Fikes to provide Planck binary for Alpine (see [this issue](https://github.com/planck-repl/build-envs/issues/1))

Reply:
The CircleCI builds that are run since yesterday use Debian. Does that mean there's no blocker now?